native/wasmex/src/module.rs

```diff
@@ -132,18 +175,6 @@ fn function_info<'a>(env: rustler::Env<'a>, ty: &FuncType) -> Term<'a> {
     make_tuple(env, &terms)
 }
 
-fn val_type_to_atom(val_type: &ValType) -> Atom {
-    match val_type {
-        ValType::I32 => atoms::i32(),
-        ValType::I64 => atoms::i64(),
-        ValType::F32 => atoms::f32(),
-        ValType::F64 => atoms::f64(),
-        ValType::V128 => atoms::v128(),
-        ValType::ExternRef => atoms::extern_ref(),
-        ValType::FuncRef => atoms::func_ref(),
-    }
-}
-
```

Comment:
thanks for making this a proper enum with a `From` impl! looks much cleaner than what I wrote